packages/client/src/views/OfficeHome/OfficeHome.tsx

```diff
@@ -369,14 +370,16 @@ export class OfficeHomeView extends React.Component<
         {this.role && NATL_ADMIN_ROLES.includes(this.role) && (
           <Redirect to={PERFORMANCE_HOME} />
         )}
-        {this.role && SYS_ADMIN_ROLES.includes(this.role) && (
-          <Redirect
-            to={{
-              pathname: PERFORMANCE_HOME,
-              search: `?locationId=${this.jurisdictionLocationId}`
-            }}
-          />
-        )}
+        {this.role &&
+          (SYS_ADMIN_ROLES.includes(this.role) ||
+            PERFORMANCE_MANAGEMENT_ROLES.includes(this.role)) && (
```

Comment:
Did you check whether a user with PERFORMANCE_MANAGEMENT_ROLES can only see his/her office performance?